### ISSUE TITLE
Node-red runtime\nodes\context removed

### ISSUE THREAD

**Issue by MParrao** [node-red/node-red#2102]
Hi,

In the past (version 0.19.6) while using node-red-node-test-helper for nodes unit tests it used certain libraries from node-red i.e.:

...\node-v8.12.0-win-x64\node_modules\node-red\red\runtime\nodes\context
...\node-v8.12.0-win-x64\node_modules\node-red\red\api\editor\comms

Now on the latest release the node-red-node-test-helper is trying to find it at:
...\node-v8.12.0-win-x64\node_modules\node-red\lib\runtime\nodes\context

However is not in there or even in the path used on version 0.19.6 also I'm not able to find them manually. Just double checking if this was done on purpose

**Comment by knolleary** [comment]
Hi - I forgot to publish the updated `node-red-node-test-helper` to npm that knows how to handle the 0.20 updates.

We can't immediately publish the update to npm for boring admin reasons - the person with npm publish rights on that module isn't available at the moment. Once that's sorted I'll update this issue when version `0.2.0` of the module has been published,

In the mean time, if you change your dependencies entry in package.json to:

```
    "node-red-node-test-helper": "github:node-red/node-red-node-test-helper",
```

Then you will get the code from git rather than npm. This is in fact what we do in the main `node-red` repo - something we really should have spotted to remind us to publish the update before the release.

**Comment by knolleary** [comment]
Now published as `0.2.0`.

If you hit any problems with it, please raise against https://github.com/node-red/node-red-node-test-helper
